### PR TITLE
PMM-7321 Alerts UI tests

### PR DESCRIPTION
**Pull request by yurkovychv**

### CODE REVIEW

File: pmm-app/tests/ia/pages/alertsPage.js [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+    if (title === 'Silence') {
+      const bgColorBeforeAction = await I.grabCssPropertyFrom(
+        `${this.elements.alertRow(alertName)}/td`,
+        'background-color',
+      );
+
+      I.click(this.buttons.silenceActivate(alertName));
+      I.verifyPopUpMessage(this.messages.successfullySilenced);
+      I.seeTextEquals('Silenced', this.elements.stateCell(alertName));
+      const bgColorAfterAction = await I.grabCssPropertyFrom(
```
Comment (puneet0191) [inline pmm-app/tests/ia/pages/alertsPage.js:43]: just move this into a new action in adminPage class, `veriryColorChange`, or something similar, make it generic to be re-used anywhere we want. As I see you already use it in two functions.

**Review by puneet0191 — APPROVED** [review]

File: pmm-app/tests/ia/pages/alertsPage.js (hunk 1 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/alertsPage.js:43]: Actually, I don't see the reason to put a single assertion into a separate function.

File: pmm-app/tests/ia/pages/api/alertsAPI.js [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+  specific number of alerts to appear
+  */
+  async waitForAlerts(timeout, expectedAlertsCount = 0) {
+    for (let i = 0; i < timeout; i++) {
+      const results = await this.getAlertsList();
+
+      // waiting for some exact number of alerts to be active
+      if (results && results.length >= expectedAlertsCount) break;
+
+      I.wait(1);
```
Comment (atymchuk) [inline pmm-app/tests/ia/pages/api/alertsAPI.js:17]: nit: this is one millisecond, right? or one second?

**Review by atymchuk — APPROVED** [review]

File: pmm-app/tests/ia/pages/api/alertsAPI.js (hunk 2 above)
Comment (yurkovychv) [reply pmm-app/tests/ia/pages/api/alertsAPI.js:17]: this is one second actually

**Review by puneet0191 — APPROVED** [review]

**Pull request merged**